ggshield/core/url_utils.py

```diff
@@ -0,0 +1,97 @@
+from urllib.parse import ParseResult, urlparse
+
+from click import UsageError
```

Comment:
We should avoid using `click` in the core layer, no? But maybe is for another commit.

Reply:
yes and yes :)